sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -39,14 +39,22 @@ internal FormField(string name, FieldValue_internal fieldValue, IReadOnlyList<Re
             Name = name;
             LabelData = null;
 
-            IReadOnlyList<FormElement> FormElement = fieldValue.Elements != null
-                ? ConvertTextReferences(fieldValue.Elements, readResults)
-                : new List<FormElement>();
+            // Bounding box, page and text are not returned by the service in two scenarios:
+            //   - When this field is global and not associated with a specific page (e.g. ReceiptType).
+            //   - When this field is a collection, such as a list or dictionary.
+            //
+            // In these scenarios we do not set a ValueData.
+
+            if (fieldValue.BoundingBox.Count != 0 || fieldValue.Page != null || fieldValue.Text != null)
+            {
+                IReadOnlyList<FormElement> FormElement = ConvertTextReferences(fieldValue.Elements, readResults);
 
-            // TODO: FormEnum<T> ?
-            BoundingBox boundingBox = fieldValue.BoundingBox == null ? default : new BoundingBox(fieldValue.BoundingBox);
+                // TODO: FormEnum<T> ?
+                BoundingBox boundingBox = new BoundingBox(fieldValue.BoundingBox);
+
+                ValueData = new FieldData(fieldValue.Text, fieldValue.Page.Value, boundingBox, FormElement);
+            }
```

Comment:
should there be an else here to assign `ValueData = null` ? or I am missing something :(

Reply:
Reference types are already null by default, so it will stay like this if we don't touch it. Do you think it's better to make it explicit?

Reply:
I prefer if we make it more obvious.  We already do it for `LabelData`, might as well do it for `ValueData`

Reply:
Done!